tests/unit/test_kwargs/test_multipart_data.py

```diff
@@ -597,3 +619,93 @@ def test_invalid_multipart_raises_client_error() -> None:
             headers={"Content-Type": "multipart/form-data; charset=utf-8; boundary=20b303e711c4ab8c443184ac833ab00f"},
         )
         assert response.status_code == HTTP_400_BAD_REQUEST
+
+
+# Test for GitHub issue #4204: Empty strings in multipart/form-data should be preserved
+class EmptyStringTestData(BaseModel):
```

Comment:
Why is this testing a Pydantic model? This issue has nothing to do with Pydantic